SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -297,7 +298,10 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 
 	UPROPERTY(EditAnywhere, config, Category = "Simulated Players", meta = (EditCondition = "bSimulatedPlayersIsEnabled", ConfigRestartRequired = false, DisplayName = "Number of simulated players"))
 		uint32 NumberOfSimulatedPlayers;
-	
+
+	UPROPERTY(EditAnywhere, Config, Category = "Region settings", meta = (ConfigRestartRequired = true, DisplayName = "Running in China"))
```

Comment:
Is `ConfigRestartRequired ` really true here?

Reply:
I vaguely remember there being a callback for changed properties so i'll investigate that because that would prevent the restart - it's because the deployment code can't read directly from the setting (used in LocalDeploymentManager)

Reply:
I replied on another thread with info here https://github.com/spatialos/UnrealGDK/pull/1531/files#r352600670

Reply:
So if I understand this correctly, a restart would also be required because we start our services on editor startup, and we don't want to over-complicate things by adding logic to restart the service in a different environment upon toggling this.

Reply:
yeah that's right - Josh suggested that we can get access to the settings once we have added service settings or something accessible by the services project on the dev branch